package.json

```diff
@@ -10,17 +10,29 @@
  "license": "GPL-2.0+",
  "private": true,
  "devDependencies": {
+  "babel-core": "^6.25.0",
+  "babel-loader": "^7.1.1",
+  "babel-plugin-transform-object-rest-spread": "^6.26.0",
+  "babel-plugin-transform-react-jsx": "^6.24.1",
+  "babel-plugin-transform-runtime": "^6.23.0",
+  "babel-preset-env": "^1.7.0",
+  "cross-env": "^5.1.5",
   "eslint": "^4.19.1",
   "eslint-config-wordpress": "^2.0.0",
-  "grunt": "^1.0.1",
+  "eslint-plugin-react": "^7.8.2",
+  "grunt": "^1.0.2",
   "grunt-contrib-clean": "^1.1.0",
   "grunt-contrib-copy": "^1.0.0",
   "grunt-contrib-jshint": "^1.1.0",
   "grunt-shell": "^2.1.0",
-  "grunt-wp-deploy": "^1.2.1"
+  "grunt-wp-deploy": "^1.2.1",
+  "timeago.js": "3.0.2",
+  "webpack": "^3.12.0"
  },
+ "main": "blocks/index.js",
  "scripts": {
-  "build": "grunt build",
-  "deploy": "grunt deploy"
+  "build": "cross-env BABEL_ENV=default webpack; grunt build",
```

Comment:
There's surely a better way do do this then I've done here. It probably needs `BABEL_ENV=production` for example, but also I wonder if [`grunt-webpack`](https://www.npmjs.com/package/grunt-webpack) should be used so that it can be called automatically as part of the `build` task.